### ISSUE TITLE
Latest release seems to have broken extension in GitHub Codespaces

### ISSUE THREAD

**Issue by picatz** [VSCodeVim/Vim#7166]
**Describe the bug**
This extension no longer seems to works in GitHub Codespaces. The newest version fails to load (`v1.21.8`).

**To Reproduce**

1.  Open a repository with GitHub Codespaces with this this extension installed.
2.  See that none of the Vim-like features work. No `—NORMAL—` or `—INSERT—` in the status bar.
3.  Go to worker extension log output.
4.  See error:

```console
[2021-10-17 12:55:06.579] [exthost.log] [info] extension host started
[2021-10-17 12:55:06.601] [exthost.log] [info] ExtensionService#_doActivateExtension vscodevim.vim, startup: true, activationEvent: '*'
[2021-10-17 12:55:06.869] [exthost.log] [error] Activating extension vscodevim.vim failed due to an error:
[2021-10-17 12:55:06.873] [exthost.log] [error] {}
```

**Expected behavior**
I’d expect this to work wonderfully, like it was before.

**Environment (please complete the following information):**

- Extension (VsCodeVim) version: `v1.21.8`
- VSCode version: `v1.61.11`
- OS: Ubuntu 20.04.3 LTS

**Additional context**

```console
$ lsb_release -a 
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.3 LTS
Release:        20.04
Codename:       focal
```

I don’t see any other related log output. However, I attempted to revert to an older version, but was unable to do so, perhaps revealing the real problem?

```console
$ code --install-extension vscodevim.vim@1.21.7
Installing extensions on Codespaces...
Cannot install the 'vscodevim.vim' extension because it is declared to not run in this setup.
```

I looked up the updates related to VSCode v1.61.11 and found a section on “platform specific extensions”, which seems related to the error I’m seeing? https://code.visualstudio.com/updates/v1_61#_platform-specific-extensions

I’m also seeing some related-ish chatter in the issue for platform specific VSCode extensions https://github.com/microsoft/vscode/issues/23251 and also found https://github.com/microsoft/vscode-vsce/issues/625

Not really sure what’s going on here, but happy to provide more information to help debug this issue further.

**Comment by J-Fields** [comment]
Duplicate of #6826

**Comment by J-Fields** [comment]
This should be fixed in 1.29.1
